plugins/chat/assets/stylesheets/common/chat-side-panel-resizer.scss

```
@@ -3,7 +3,7 @@
   bottom: 0;
 
   position: absolute;
-  z-index: calc(z("header") - 1);
+  z-index: calc(z("composer", "content") - 1);
```

🏌️ ⛳ (by using scss math)

```suggestion
  z-index: z("composer", "content") - 1;
```